src/ngraph/ngraph_emitter.cc

```diff
@@ -1552,6 +1555,16 @@ void Emitter::CreateLossOps() {
 
     return grad;
   };
+  loss_op_backward_funcs_["LinearRegressionOutput"] = [this](
+      const NodePtr& node, const NgraphNodePtr& adjoint) {
+    auto label = op_map_[node->inputs_[0]];
+    auto data = op_map_[node->inputs_[1]];
```

Comment:
One bug we hit for softmax is the data and label shape can be different. Could you verify this handles the case like https://github.com/NervanaSystems/ngraph-mxnet/blob/c02047ab46aaf2cc2a8ad2cb1c59c46eaa2d59ef/src/operator/regression_output-inl.h#L62